modules/pytket-qiskit/tests/backend_test.py

```diff
@@ -860,3 +868,84 @@ def test_compilation_correctness() -> None:
         m_inv_fin = lift_perm(inv_fin)
 
         assert compare_unitaries(u, m_inv_fin @ compiled_u @ m_ini)
+
+
+# pytket-extensions issue #78
+def test_symbolic_rebase() -> None:
+    circ = QuantumCircuit(2)
+    circ.rx(Parameter("a"), 0)
+    circ.ry(Parameter("b"), 1)
+    circ.cx(0, 1)
+
+    pytket_circ = qiskit_to_tk(circ)
+
+    # rebase pass could not handle symbolic parameters originally and would fail here:
+    _rebase_pass.apply(pytket_circ)
+
+
+def _tk1_to_rotations(a: float, b: float, c: float) -> Circuit:
+    """Translate tk1 to a RzRxRz so AerUnitaryBackend can simulate"""
+    circ = Circuit(1)
+    circ.Rz(c, 0).Rx(b, 0).Rz(a, 0)
+    return circ
+
+
+def _verify_single_q_rebase(
+    backend: AerUnitaryBackend, a: float, b: float, c: float
+) -> bool:
+    """Compare the unitary of a tk1 gate to the unitary of the translated circuit"""
+    rotation_circ = _tk1_to_rotations(a, b, c)
+    u_before = backend.get_unitary(rotation_circ)
+    circ = Circuit(1)
+    circ.add_gate(OpType.TK1, [a, b, c], [0])
+    _rebase_pass.apply(circ)
+    u_after = backend.get_unitary(circ)
+    return np.allclose(u_before, u_after)
+
+
+testvals = [
+    (1.5, 0, 2.5),
+    (1.5, 2, 2.5),
+    (1.5, 4, 2.5),
+    (0, -0.5, 0),
+    (0, 1.5, 0),
+    (0, 3.5, 0),
+    (2, -0.5, 0),
+    (2, 1.5, 0),
+    (2, 3.5, 0),
+    (0, -0.5, 2),
+    (0, 1.5, 2),
+    (0, 3.5, 2),
+    (2, -0.5, 2),
+    (2, 1.5, 2),
+    (2, 3.5, 2),
+    (2, 1, 0),
+    (2, -1, 0),
+    (2, 3, 0),
+    (4, 1, 0),
+    (4, -1, 0),
+    (4, 3, 0),
+    (2, 1, 8),
+    (2, -1, 8),
+    (2, 3, 8),
+    (1, 1, 0),
+    (3, -1, 0),
+    (7, 3, 0),
+    (3.5, 1, 0),
+    (0, -1, 3.7),
+    (0, 3, 1),
+    (0.5, 1.3, 0.5),
+    (2.5, 1.3, 0.5),
+    (0.5, 1.3, 2.5),
+    (2.5, 1.3, 2.5),
+    (-1.5, 1.3, -1.5),
+    (7.5, 1.3, 1.5),
+    (3.5, 7.7, 0.5),
+    (8.5, 1.3, 2.5),
+]
+
+
+@pytest.mark.parametrize("a,b,c", testvals)
+def test_rebase_phase(a: float, b: float, c: float) -> None:
+    backend = AerUnitaryBackend()
+    assert _verify_single_q_rebase(backend, a, b, c)
```

Comment:
```suggestion
def test_rebase_phase() -> None:
    backend = AerUnitaryBackend()
    for a in [0.6, 0, 1, 2, 3]:
        for b in [0.7, 0, 0.5, 1, 1.5]:
            for c in [0.8, 0, 1, 2, 3]:
                assert _verify_single_q_rebase(backend, a, b, c)
```
This would give me greater confidence that we've covered all bases, if this should be refactored in the future.